src/patternfly/components/Drawer/drawer.scss

```diff
@@ -270,22 +264,21 @@ $pf-v5-c-drawer__panel--list--width: (25, 33, 50, 66, 75, 100);
   z-index: var(--#{$drawer}__content--ZIndex);
   flex-basis: var(--#{$drawer}__content--FlexBasis);
   order: 0;
-
-  // background-color: var(--#{$drawer}__content--BackgroundColor);
+  background-color: var(--#{$drawer}__content--BackgroundColor);
```

Comment:
@lboehling noticed in some of the full page examples, we've lost the rounded corners around the page main content area. For example, from https://patternfly-pr-6183.surge.sh/components/drawer/html-demos/expanded-bottom/

<img width="989" alt="Screenshot 2024-01-12 at 4 34 48 PM" src="https://github.com/patternfly/patternfly/assets/35148959/349a98e4-e1ef-468f-90b5-53e5ba66ad12">

The rounded corners are on `.pf-v5-c-page__main`, but in these full page examples, `.pf-v5-c-page__main` is a child of `.pf-v5-c-drawer__content`, which has this background color added and keeps the corners on page__main from showing up. Opened https://github.com/patternfly/patternfly/issues/6225 to try and address that, since we probably need to test a bit and see what the ideal solution is there.

Reply:
Thanks for filing this issue @mcoker 
Lots of products are using drawer in this way so definitely important to fix.